crates/bevy-inspector-egui/Cargo.toml

```diff
@@ -8,39 +8,55 @@ license = "MIT OR Apache-2.0"
 description = "Inspector plugin for the bevy game engine"
 documentation = "https://docs.rs/bevy-inspector-egui"
 keywords = ["bevy", "inspector", "egui", "editor", "reflect"]
-categories = ["game-development", "gui", "visualization", "development-tools::procedural-macro-helpers"]
+categories = [
+    "game-development",
+    "gui",
+    "visualization",
+    "development-tools::procedural-macro-helpers",
+]
 
 [features]
 default = ["bevy_pbr"]
 highlight_changes = []
 
 [dependencies]
-bevy-inspector-egui-derive = { version = "0.18.0", path = "../bevy-inspector-egui-derive" }
-bevy_app = { version = "0.10" }
-bevy_asset = { version = "0.10" }
-bevy_core = { version = "0.10" }
-bevy_ecs = { version = "0.10" }
-bevy_hierarchy = { version = "0.10" }
-bevy_log = { version = "0.10" }
-bevy_math = { version = "0.10" }
-bevy_reflect = { version = "0.10" }
-bevy_utils = { version = "0.10" }
-bevy_window = { version = "0.10" }
-
-bevy_core_pipeline = { version = "0.10" }
-bevy_pbr = { version = "0.10", optional = true }
-bevy_render = { version = "0.10" }
+bevy-inspector-egui-derive = { version = "0.18.1", path = "../bevy-inspector-egui-derive" }
+bevy_app = { version = "0.11" }
+bevy_asset = { version = "0.11" }
+bevy_core = { version = "0.11" }
+bevy_ecs = { version = "0.11" }
+bevy_hierarchy = { version = "0.11" }
+bevy_log = { version = "0.11" }
+bevy_math = { version = "0.11" }
+bevy_reflect = { version = "0.11" }
+bevy_utils = { version = "0.11" }
+bevy_window = { version = "0.11" }
+
+bevy_core_pipeline = { version = "0.11" }
+bevy_pbr = { version = "0.11", optional = true }
+bevy_render = { version = "0.11" }
 
 egui = "0.21"
-bevy_egui = "0.20"
+bevy_egui = { git = "https://github.com/Vrixyz/bevy_egui.git", branch = "bevy-main" }
```

Comment:
Worth noting that to avoid future breakage, this PR should either be delayed until upstream bevy_egui has been fully released, or pinned to a specific commit

Reply:
Agreed. Keeping this PR open until https://github.com/mvlabat/bevy_egui/pull/188 gets accepted and released.

Reply:
https://github.com/mvlabat/bevy_egui/pull/188 is merged to `main` but yet to be released.